spec/jobs/set_catkeys_and_barcodes_job_spec.rb

```diff
@@ -92,14 +92,14 @@
 
       it 'attempts to update the catkey for each druid with an empty array and not nil' do
         expect(subject).to receive(:with_bulk_action_log).and_yield(buffer)
-        expect(subject).to receive(:update_catkey_and_barcode).with(change_set1, { catkeys: [] }, buffer) # changed to empty
-        expect(subject).not_to receive(:update_catkey_and_barcode).with(change_set2, { catkeys: [] }, buffer) # not changed
-        expect(subject).to receive(:update_catkey_and_barcode).with(change_set3, { catkeys: ['44444'] }, buffer) # changed to a value
+        expect(subject).to receive(:update_catkey_and_barcode).with(change_set1, { catkeys: [], refresh: true }, buffer) # changed to empty
+        expect(subject).not_to receive(:update_catkey_and_barcode).with(change_set2, { catkeys: [], refresh: true }, buffer) # not changed
+        expect(subject).to receive(:update_catkey_and_barcode).with(change_set3, { catkeys: ['44444'], refresh: true }, buffer) # changed to a value
         subject.perform(bulk_action_no_process_callback.id, params)
         expect(bulk_action_no_process_callback.druid_count_total).to eq druids.length
-        expect(change_set1).to have_received(:validate).with(catkeys: []) # set to empty
-        expect(change_set2).to have_received(:validate).with(catkeys: []) # not changed
-        expect(change_set3).to have_received(:validate).with(catkeys: ['44444']) # changed to a value
+        expect(change_set1).to have_received(:validate).with(catkeys: [], refresh: true) # set to empty
+        expect(change_set2).to have_received(:validate).with(catkeys: [], refresh: true) # not changed
+        expect(change_set3).to have_received(:validate).with(catkeys: ['44444'], refresh: true) # changed to a value
```

Comment:
should there be a spec for `refresh: false`?

Reply:
Refresh can't be set from the plain-old set_catkeys_and_barcodes_job invocation, which this is testing.